charts/galoy/templates/cronjob.yaml

```diff
@@ -5,9 +5,15 @@ metadata:
   name: {{ .name }}
   labels:
     app.kubernetes.io/managed-by: Helm
+    app: {{ .name }}
 spec:
   schedule: {{ .schedule }}
   jobTemplate:
+    metadata:
+      {{- with .podLabels }}
```

Comment:
This should not be exposed. The network policy related labels should be hard coded.

Reply:
Fixed